runtime/hydradx/src/assets.rs

```diff
@@ -429,6 +431,54 @@ impl pallet_dca::Config for Runtime {
 	type WeightInfo = weights::dca::HydraWeight<Runtime>;
 }
 
+pub struct AmmWeights;
+impl AmmWeights {
+	pub fn sell_overhead_weight() -> Weight {
+		weights::route_executor::HydraWeight::<Runtime>::sell_in_lbp()
+			.saturating_sub(weights::lbp::HydraWeight::<Runtime>::trade_execution_sell())
+	}
+
+	pub fn buy_overhead_weight() -> Weight {
+		weights::route_executor::HydraWeight::<Runtime>::buy_in_lbp()
+			.saturating_sub(weights::lbp::HydraWeight::<Runtime>::trade_execution_buy())
+	}
+}
+impl AmmTradeWeights<AssetId> for AmmWeights {
+	fn sell_weight(route: &[Trade<AssetId>]) -> Weight {
+		let mut weight = Weight::zero();
+
+		for trade in route {
+			let amm_weight = match trade.pool {
+				PoolType::Omnipool => weights::omnipool::HydraWeight::<Runtime>::trade_execution_sell(),
+				PoolType::LBP => weights::lbp::HydraWeight::<Runtime>::trade_execution_sell(),
+				PoolType::Stableswap(_) => Weight::zero(),
```

Comment:
instead of zero, how about putting some default weight of Omnipool or LBP, as a safeguard? 

Let's say that someone adds router support for XYK or stableswap, but forgets to change this, so the weight will remain zero. Although it has a small chance, we never know, better safe than sorry.

Also, what I have done on my branch is to add router integration tests returning "NotSupported" error for the AMMs not supported yet, and also also placing comments there to be sure that these weights are updated by the one who adds support for a new AMM.

Reply:
replaced by the omnipool weights.